<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mdtSerialPortConfigWidget</class>
 <widget class="QWidget" name="mdtSerialPortConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lbBaudRate">
     <property name="text">
      <string>Baud rate</string>
     </property>
     <property name="buddy">
      <cstring>cbBaudRate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="cbBaudRate"/>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QGroupBox" name="gbDataBitCount">
     <property name="title">
      <string>Data bits</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rbDataBit5">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbDataBit6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbDataBit7">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbDataBit8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="gbStopBitCount">
     <property name="title">
      <string>Stop bits</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="rbStopBit1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbStopBit2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="gbParity">
     <property name="title">
      <string>Parity</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbParityNone">
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbParityEven">
        <property name="text">
         <string>Even</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbParityOdd">
        <property name="text">
         <string>Odd</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="3" rowspan="2">
    <widget class="QGroupBox" name="gbFlowCtl">
     <property name="title">
      <string>Flow control</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="rbFlowCtlNone">
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbFlowCtlXonXoff">
        <property name="text">
         <string>Xon/Xoff</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lbXonChar">
          <property name="text">
           <string>Xon char</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbXonChar">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lbXoffChar">
          <property name="text">
           <string>Xoff char</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbXoffChar">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="rbFlowCtlRtsCts">
        <property name="text">
         <string>RTS/CTS</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>43</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbBaudRate</tabstop>
  <tabstop>rbDataBit5</tabstop>
  <tabstop>rbDataBit6</tabstop>
  <tabstop>rbDataBit7</tabstop>
  <tabstop>rbDataBit8</tabstop>
  <tabstop>rbStopBit1</tabstop>
  <tabstop>rbStopBit2</tabstop>
  <tabstop>rbParityNone</tabstop>
  <tabstop>rbParityEven</tabstop>
  <tabstop>rbParityOdd</tabstop>
  <tabstop>rbFlowCtlNone</tabstop>
  <tabstop>rbFlowCtlXonXoff</tabstop>
  <tabstop>rbFlowCtlRtsCts</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
